src/auditor/dynamic_linkage.jl

```diff
@@ -326,6 +330,7 @@ function patchelf_flags(p::AbstractPlatform)
     flags = []
 
     # ppc64le and aarch64 have 64KB page sizes, don't muck up the ELF section load alignment
+    # TODO: What is the riscv64 page size?
```

Comment:
@staticfloat maybe you have a clue of what this should be?

Reply:
Wikipedia says it's 4k. But then, it's also 4k for aarch64...

Reply:
FTR, the “authoritative” source on this that I like to use is gold from binutils: https://github.com/search?q=repo%3Abminor%2Fbinutils-gdb%20abi_pagesize&type=code but that doesn’t have anything about riscv for some reason. I think it’s most likely 4k since the only reason to have larger page sizes is for higher memory counts, which is, AFAIK, not common with riscv.

Reply:
arm64 supports 64 kB page sizes. It's reasonable to use these in an executable. As far as I know, riscv64 supports only 4 kB and 2 MB (and larger) page sizes, and it wouldn't be reasonable to use these in an executable. (x86_64 doesn't do this either.)